src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -24755,7 +24772,6 @@ Map {
     "render": [Function],
   },
   "determineCardRange" => Object {},
-  "determineMaxValueCardAttributeCount" => Object {},
```

Comment:
Is this a breaking change? or is this provided by the component?

Reply:
Theres technically no 'max' number of attributes for ValueCard with the redesign, but it is a breaking change thats its no longer being exported. I can leave it in with a deprecation warning

Reply:
I re-added the export and added a deprecation warning when in `__DEV__`